# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/iostat/usr-loca-prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- iostat-2.0/Makefile	2004-01-06 23:35:49.000000000 +0100
+++ iostat-2.0-fix/Makefile	2004-01-29 10:34:56.000000000 +0100
@@ -5,6 +5,8 @@
 CFLAGS = -g -O2 -Wall -fomit-frame-pointer
 LDFLAGS =
 
+prefix = /usr/local
+
 SRC = iostat.c
 BIN = iostat
 
@@ -14,9 +16,9 @@
 	$(CC) $(CFLAGS) $(SRC) -o $@ $(LDFLAGS)
 
 install: $(BIN)
-	install -d /usr/local/bin /usr/local/man/man8
-	install -o root -g root -m 755 -s iostat /usr/local/bin
-	install -o root -g root -m 644 iostat.8 /usr/local/man/man8
+	install -d $(prefix)/bin $(prefix)/man/man8
+	install -o root -g root -m 755 -s iostat $(prefix)/bin
+	install -o root -g root -m 644 iostat.8 $(prefix)/man/man8
 
 clean:
 	rm -f $(BIN) core
